cli/unregister/unregister.go

```diff
@@ -34,6 +38,13 @@ func DoIt(forceUnregister, removeNodeUnregister bool, deepClean bool, timeout in
 	horDevice := api.HorizonDevice{}
 	cliutils.HorizonGet("node", []int{200}, &horDevice, false)
 
+	if deepClean {
+		err := backupEventLogs()
+		if err != nil {
+			msgPrinter.Printf("Cannot backup eventlogs: %v", err)
```

Comment:
Please add  `msgPrinter.Println()` here to print out a newline.

Reply:
Added printing of new line for both of `Printf` usages